lib/ffmpeg/encoderbase.py

```diff
@@ -87,6 +87,9 @@ def gen_output_opts(self):
     if vars(self).get("vforced_idr", None) is not None:
       opts += " -force_key_frames expr:1 -forced_idr 1"
 
+    if self.codec.startswith("avc") and self.rcmode in ["cbr", "vbr", "cbr_lp", "vbr_lp"]:
+      opts += " -int_ref_type 1 -int_ref_cycle_size 4 -int_ref_cycle_dist 20"
+
```

Comment:
1. There is no `cbr_lp` or `vbr_lp` for `self.rcmode`.  The lp is only captured in `self.lowpower`.  Thus, only need to check for `cbr` and `vbr` if want to apply to both vme and vdenc.
2. These properties are only valid for ffmpeg-qsv.  This adds them for all ffmpeg plugins, which is not correct.
3. Both AVC and HEVC support these properties in ffmpeg-qsv.  So why only AVC enabled?
4. This is essentially the same hack attempted earlier in the config file patch, just in another disguise.  As explained there, this adds these (hard-coded) parameters for all test cases in these categories.  Thus, this changes pre-existing cases and invalidates everyone's baseline refs for all test suites (quality, sanity, bat, full,...).  This is not acceptable.

Please create a new test for intra refresh and configure separate cases for them in the test configs.  Look at `test/ffmpeg-qsv/encode/avc.py::forced_idr` test for an example.  And the test function should include the intra refresh parameters that will be configured by user test config/spec.